lib/kit.js

```diff
@@ -44,6 +44,7 @@ class CoinsenceKit {
 
     return this.contract.functions.newInstance(spaceName, descHash, coinName, coinSymbol, coinDecimals, root, callOptions)
       .then(transaction => {
+        
         return transaction.wait().then(result => {
```

Comment:
we must get rid of this `wait()` here - otherwise the function resolves not earlier and we still wait for a confirmation.